pipenv/vendor/vendor.txt

```diff
@@ -31,7 +31,7 @@ toml==0.10.2
 tomli==1.1.0
 tomlkit==0.9.2
 urllib3==1.26.6
-vistir==0.5.2
+vistir==0.5.2  # Deprecated: our copy is modified to fix bugs
```

Comment:
we should try commenting this out entirely and see if vendoring still passes -- I think it won't though.  Future vendoring would overwrite this bugfix.  Maybe we should move vistir out of the vendor namespace and chip away at it from within `pipenv` module at same level as `utils`?

Reply:
If we can remove vistir completely, I will be extremely happy. But I am afraid it won't be so easy.
However, can you add a patch in the patches directory?

Reply:
I'd prefer to not patch it and move it out of vendor into a the level of utils so we can whack away at it.

Reply:
Or -- I just got access to all of https://github.com/sarugaku so I can upstream this as well.

Reply:
That's what I meant. I wrote uranusjar today and he was very quick to respond.